apps/docs/src/lib/mdx/components/MdxFileView/customComponents.tsx

```diff
@@ -89,19 +89,27 @@ export const customComponents = {
 
   a: ({ children, href }: PropsWithChildren<{ href?: string }>) => {
     if (href?.startsWith("http")) {
+      const url = new URL(href);
+
       return (
-        <Link href={href} inline>
+        <Link
+          href={href}
+          inline
+          target={
+            url.origin === "https://mittwald.github.io" ? undefined : "_blank"
```

Comment:
origin is the origin url of the target url, this will always be the domain the app is running from. you need `hostname`

eg.
```suggestion
            url.hostname === "mittwald.github.io" ? undefined : "_blank"
```